ui/apps/platform/src/Containers/Violations/Details/Deployment/DeploymentDetails.tsx

```diff
@@ -144,14 +92,21 @@ const DeploymentDetails = ({ alertDeployment }: DeploymentDetailsProps) => {
                     <FlexItem>
                         <Card isFlat aria-label="Port configuration">
                             <CardBody>
-                                {relatedDeploymentPorts.length > 0
-                                    ? formatDeploymentPorts(relatedDeploymentPorts).map(
-                                          (port, idx) => (
-                                              // eslint-disable-next-line react/no-array-index-key
-                                              <ObjectDescriptionList data={port} key={idx} />
-                                          )
-                                      )
-                                    : 'None'}
+                                {relatedDeploymentPorts.length === 0
+                                    ? 'None'
+                                    : relatedDeploymentPorts.map((port, i) => {
+                                          /* eslint-disable react/no-array-index-key */
+                                          return (
+                                              <React.Fragment key={i}>
```

Comment:
Dave, can you recommend a unique composite key from port properties?

Reply:
In this case I'm honestly not sure. My feeling is that `containerPort` plus container **name** or **id** would be ideal, but the response doesn't supply either of those items. We could do `name` plus `containerPort`, but it seems likely there is a small chance of a multi-container deployment that will result in duplicate keys.

In this case, the safest bet might be to just use the index like you are doing now. These items are immutable and will not be updated or re-ordered, so using the index should be reliable for what we need it for here.